packages/wallet/frontend/src/components/providers/index.tsx

```diff
@@ -11,19 +11,19 @@ type AppProviderProps = {
 
 export const AppProvider = ({ children }: AppProviderProps) => {
   return (
-    <ThemeProvider
-      attribute="class"
-      defaultTheme="light"
-      enableSystem={false}
-      themes={['light', 'dark']}
-    >
-      <MenuProvider>
-        <OnboardingProvider>
-          <PasswordProvider>
-            <DialogProvider>{children}</DialogProvider>
-          </PasswordProvider>
-        </OnboardingProvider>
-      </MenuProvider>
-    </ThemeProvider>
+    // <ThemeProvider
```

Comment:
you want to keep the comments?

Reply:
That was my initial plan, but now.. No, not going to keep them.